libs/brig-types/src/Brig/Types/Team/LegalHold.hs

```diff
@@ -162,3 +162,28 @@ instance FromJSON UserLegalHoldStatus where
       "pending" -> pure UserLegalHoldPending
       "disabled" -> pure UserLegalHoldDisabled
       x -> fail $ "unexpected status type: " <> T.unpack x
+
+data LegalHoldClientRequest =
+    LegalHoldClientRequest
+    { lhcrRequester  :: !UserId
+    , lhcrTargetUser :: !UserId
+    , lhcrLastPrekey :: !LastPrekey
+    , lhcrPrekeys    :: ![Prekey]
+    } deriving stock (Eq, Show)
```

Comment:
I'll need Generic here for servant-swagger, but I can add that when I get to it.